/*
This is the python version proto3 source of DetectionService.
*/

syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.detection";
option java_outer_classname = "DetctionProto";
option objc_class_prefix = "HLW";

package detection;

service Detection {
    rpc Predict (DetectionRequest) returns (DetectionResponse) {}
}

message DetectionRequest {
    bytes originImage = 1;
}

message DetectionResponse {
    bytes predictedImage = 1;
    repeated int32 predictedResults = 2;
}
